integration_tests/run_pyspark_from_build.sh

```diff
@@ -129,6 +129,9 @@ else
     export PYSP_TEST_spark_ui_showConsoleProgress='false'
     export PYSP_TEST_spark_sql_session_timeZone='UTC'
     export PYSP_TEST_spark_sql_shuffle_partitions='12'
+    # prevent cluster shape to change - and fail quicker rather than retry
+    export PYSP_TEST_spark_task_maxFailures='1'
```

Comment:
these aren't getting applied when using the spark-submit way to run , correct?  Seems like we should be consistent

Reply:
Nice catch. Are `jenkins/spark-tests.sh` and then `run_pyspark_from_build.sh` the only two places that trigger the tests? If so I'd need to change spark-tests.sh.

Reply:
I think those are the only 2 in this repo